kotlin-jvm/src/main/kotlin/playground/Clikt.kt

```diff
@@ -0,0 +1,25 @@
+@file:Suppress("PackageDirectoryMismatch")
+
+package playground.clikt
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.*
+
+
+/**
+ * ajalt/Clikt : Multiplatform command line interface parsing for Kotlin
+ *
+ * - [Website](https://ajalt.github.io/clikt/)
+ * - [Github](https://github.com/ajalt/clikt)
+ */
+class CliktPlayground : CliktCommand() {
+    private val language: String by option(help="Language of greeting").default("EN")
+    private val greeting: String by option(help="Greeting").default("Hello")

```

Comment:
There should be spaces around the `=` signs. Next time, please do "Reformat code" in the IDE (shortcut is ctrl/cmd + alt/option + L).

@jmfayard Can you review the code style/formatting for next PRs?